### DISCUSSION TITLE
Why does `next/og` only work on the edge when `@vercel/og` works perfectly fine in Node?

### DISCUSSION THREAD

**Discussion by akshatmittal** [vercel/next.js#60003] [App Router]
Basically title.

In order to use `ImageResponse` from `next/og`, you need to set the route or page to use the `edge` runtime and it complains if you don't do it. I don't quite understand why, especially since `@vercel/og` works perfectly fine in `nodejs` runtime. 

I literally just replaced the `ImageResponse` object to be imported from `@vercel/og` instead of `next/og` and everything just works.

Any reason, technical or otherwise, on why this limitation exists?

**Comment by septem1997** [comment]
it looks like the docs have removed the description of having to use the edge runtime, I'm using nodejs now, and it works well
